### ISSUE TITLE
Uploaded Emotes won't show up in channels

### ISSUE THREAD

**Issue by Valsis-sf** [night/betterttv#4108]
I've recently added emotes to my BTTV setup and I've noticed that, so far, the uploaded content isn't available in other streams.

If I go to the Emote Menu, I still have my old and removed emotes displayed instead of the newer ones however, if I go to my own channel, the emotes are displayed correctly. Old ones removed and replaced with the new ones recently uploaded.

I also want to note that, one of the emotes that is marked as "shared" doesn't show up in other channels either, while the two old ones that have been replaced already, are still showing.

TL:DR
Old emotes still showing after being replaced with new ones and new ones do not show up in other channels.

**Comment by night** [comment]
Emotes you manage on BetterTTV's website only correspond to you and your channel. Other broadcasters manage the emotes available in their respective channels.
